src/utils/getFileExtension.js

```diff
@@ -0,0 +1,11 @@
+/** @param {File | { name: string }} file */
+export const getFileExtension = file => {
+  if (file && file.name) {
+    const fileExtensionRegExp = /\.(\w+)$/;
+    const [extension] = (file.name.match(fileExtensionRegExp) || []);
+
+    return extension.toLowerCase();
```

Comment:
`const [extension] = (file.name.match(fileExtensionRegExp) || []);`

Theoretically `extension` can become `undefined` in case match ended up to be `null`. In that case, calling `toLowerCase` would end up in error.